### ISSUE TITLE
py3.10 c++ issue

### ISSUE THREAD

**Issue by dragonyanglong** [diffpy/diffpy.pdfgui#93]
error:
```
SystemError: PY_SSIZE_T_CLEAN macro must be defined for '#' formats
```

possible solution:
"* "The PY_SSIZE_T_CLEAN macro must now be defined to use PyArg_ParseTuple() and Py_BuildValue() formats which use #: es#, et#, s#, u#, y#, z#, U# and Z#. See Parsing arguments and building values and the PEP 353."

**Comment by dragonyanglong** [comment]
this error should be fixed in pdffit2 package, not related with pdfgui. link with this issue (https://github.com/diffpy/diffpy.pdffit2/issues/14)

**Comment by bobleesj** [comment]
We can close this since it's a pdffit2 problem and has been resolved in the PR linked above.

**Comment by bobleesj** [comment]
@sbillinge Could you please close this? The issue above by @dragonyanglong has been fixed with a PR.

**Comment by sbillinge** [comment]
closed in the same PR as #103
